lib/ParserHelpers.js

```diff
@@ -3,64 +3,75 @@
 	Author Tobias Koppers @sokra
 */
 "use strict";
+const path = require("path");
+
 const BasicEvaluatedExpression = require("./BasicEvaluatedExpression");
 const ConstDependency = require("./dependencies/ConstDependency");
 const UnsupportedFeatureWarning = require("./UnsupportedFeatureWarning");
 
-const ParserHelpers = exports;
-
-ParserHelpers.addParsedVariableToModule = function(parser, name, expression) {
-	if(!parser.state.current.addVariable) return false;
-	var deps = [];
-	parser.parse(expression, {
-		current: {
-			addDependency: function(dep) {
-				dep.userRequest = name;
-				deps.push(dep);
-			}
-		},
-		module: parser.state.module
-	});
-	parser.state.current.addVariable(name, expression, deps);
-	return true;
-};
+module.exports = class ParserHelpers {
```

Comment:
ParserHelpers is not a real class. Stay with the `exports.addParsedVaraibleToModule` style. It's more aligned with the future style `export function addParsedVariableToModule()`.